README.md

```diff
@@ -5,36 +5,21 @@
 
 It allows developers to quickly and in a simple manner define and create large number of objects whose attributes have randomly selected values from the configured set.
 
-It can be used for following:
+It can be used for:
 
 - quickly populate the database with meaningful values
 - create data based on defined rules (e.g. create 100 users out of which 10 have first name 'John' and they are born in 1980) in order to create test data for automated unit and integration tests
 - have meaningful data for load testing
+- as a data source for custom tool of your choice if you implement integration
 
-```java
-ObjectGenerator<User> userGenerator = new ObjectGenerator.Builder<User>(User.class)
-  .withValues("username", "destroyerOfW0rldz", "only_lol_catz", "aragorn_the_gray")
-  .withValues("firstname", "Alice", "Bob", "Charlie", "David")
-  .withValues("lastname", "Zed", "Yvette","Xavier")
-  .withRanges("numberOfCards", 1L, 5L)
-  .withRanges("accountBalance", 2.72, 2.73)
-  .withSubList("favoriteMovies", "Predator")
-  .withSubSet("nicknames", "al", "billie", "gray")
-  .withRanges("birthDate", LocalDateTime.of(1975, 1, 1, 0, 0), LocalDateTime.of(2001, 1, 1, 0, 0))
-  .withObjectGenerator("address", addressGenerator)
-  .toBeGenerated(1000).build();
-
-AggregatedObjectGenerator<User> aggregatedUserGenerator = new AggregatedObjectGenerator.Builder<>()
-  .withObjectGenerator(userGenerator).build();
-List<User> users = aggregatedUserGenerator.generateAll();
-```
+# Quick guide
 
-It can be used as a Java library, programatically in unit and integration tests, and from the command line (though the last one is not yet implemented).
+If you don't like to read documentation first, just go through this quick guide and try out Ranger.
```

Comment:
IMO there's no need for: "If you don't like to read...". I would just provide the link to the API docs somewhere in this document, so the reader can choose on his own.